src/api/firebase.js

```diff
@@ -133,3 +125,99 @@ export async function matchToken(listId) {
 	const jointListTokenSnapshot = await getDocs(jointListTokenQuery);
 	return jointListTokenSnapshot;
 }
+
+export function comparePurchaseUrgency(items) {
+	const overdueList = [];
+	const soonList = [];
+	const kindOfSoonList = [];
+	const notSoonList = [];
+	const purchasedList = [];
+	const inactiveList = [];
+
+	items.forEach((item) => {
+		const differenceInDays = item.dateLastPurchased
+			? getDaysBetweenDates(item.dateLastPurchased, item.dateNextPurchased)
+			: getDaysBetweenDates(item.dateCreated, item.dateNextPurchased);
+
+		if (differenceInDays >= 60) {
+			item.isInactive = true;
+			item.urgency = 'Inactive';
```

Comment:
What do you want the condition to be to make the item Inactive? 

When there is no `item.dateLastPurchased` that means the item has never been purchased. Therefore, your days between `dateCreated` and `dateNextPurchased` will be the default value selected from `AddItem` (7, 14, or 30).
It could not be over 60.